src/index.html

```diff
@@ -3,21 +3,21 @@
   <head>
     <meta charset="utf-8">
     <title>Fresh Picks</title>
-    <link rel="stylesheet" type="text/css" href="../styles.css">
+    <!-- <link rel="stylesheet" type="text/css" href="./styles.css"> -->
```

Comment:
Seems to be working without this. We could hold on to it for further down the line, just in case?

Reply:
That's not a bad idea until we figure out a little more about sass